datumaro/plugins/vgg_face2_format.py

```diff
@@ -82,10 +83,7 @@ def _load_categories(self):
             for line in lines:
                 objects = line.split()
                 label = objects[0]
-                class_name = None
-                if 1 < len(objects):
-                    class_name = objects[1]
-                label_cat.add(label, parent=class_name)
+                label_cat.add(label)
```

Comment:
Why did you decide to fully remove parent labels for VGG Face 2 format?

Reply:
I removed it because it's not the parent of this label. As I understand it, this is the name of the label and its identifier.